host_core/lib/host_core/actors/actor_supervisor.ex

```diff
@@ -53,7 +53,7 @@ defmodule HostCore.Actors.ActorSupervisor do
          ) do
       {:error, err} ->
         Logger.error("Failed to download OCI bytes for #{oci}")
-        {:stop, err}
+        {:error, err}
```

Comment:
yeah this is crazy that we didn't do anything about this earlier. Returning `{:stop, ...}` actually stops the supervisor, thereby preventing any other actor from being able to start on this host.

Reply:
Let's hope that we only returned `{:stop` to a place that didn't do that 😮